core/tests/webdriverio_utils/AdminPage.js

```diff
@@ -185,6 +251,31 @@ var AdminPage = function() {
     await this.saveChangeOfFeature(featureElement);
   };
 
+  this.enableFeatureForDev = async function(featureElement) {
+    await this.removeAllRulesOfFeature(featureElement);
+
+    await action.click(
+      'Add feature rule button',
+      featureElement
+        .$(addFeatureRuleButtonLocator)
+    );
+
+    await waitFor.visibilityOf(
+      featureElement.$(valueSelectorLocator),
+      'Value Selector takes too long to appear'
+    );
+    await (featureElement.$(valueSelectorLocator)).selectByVisibleText(
+      'Enabled');
```

Comment:
I know you used it previously to select an option element. Bu does options appear in this case. Also, I think we are giving input [here ](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/AdminPage.js#L237)and not selecting any option.

Reply:
The element we are dealing with here is a dropdown. So we need to use `selectByVisibleText` instead of setValue. In protractor sendKeys can work on dropdown but not in webdriverio.